src/hook/index.tsx

```diff
@@ -189,7 +189,7 @@ export const useRemixForm = <T extends FieldValues>({
   );
 
   const handleSubmit = useMemo(
-    () => (e: FormEvent<HTMLFormElement>) => {
+    () => (e?: FormEvent<HTMLFormElement>) => {
```

Comment:
as I said in the issue, this matches with this interface https://github.com/react-hook-form/react-hook-form/blob/52a09085c5bdb14964a146ff08862909a68de07c/src/types/form.ts#L637